data-raw/sampleData.R

```diff
@@ -24,5 +24,7 @@ rm(sampleResults)
 
 sampleSummary <- read.csv("data-raw/sampleSummary.csv")
 sampleSummary$GroupID <- as.character(sampleSummary$GroupID)
+sampleSummary$SnapshotDate <- as.Date(sampleSummary$SnapshotDate,
```

Comment:
i don't think we want sampleSummary to have a snapshot date- this is the output of `Summarize()` before it gets transformed for reporting. #1661 updates the sampleData file per #1651

Reply:
Yeah, this was to get things working-enough before #1661. I think we should get that and the test-fixing PR merged and then I can merge them into here and resolve conflicts.

Reply:
At this point in the lifecycle, this line of code is needed to get tests to pass. Can we merge this PR into dev, and then merge dev into #1661 and reconcile? I can do the conflict-resolution, I'd just really like to get all checks back to green and keep them there going forward!